### ISSUE TITLE
Deploy to web UI Mod to app engine?

### ISSUE THREAD

**Issue by onethings** [vitalidze/traccar-web#790]
Hi,
      I have considered for our local low internet connection and want deploy web UI Mod to app engine https://cloud.google.com/appengine/docs/java/tools/maven  . what files I can edit to deploy to app engine. Although I have run traccar on Google cloud platform with Ubuntu and use near location of Google data center, also using cloud DNS and using sub domain with aws cloudfront for some js and CSS. When internet connection is good, open traccar home page can reduce within 3 second.  Thanks.

**Comment by vitalidze** [comment]
From my understanding in theory you can deploy web UI Mod on app engine. But, the backend part, which collects data from devices will not work there because google platform restricts usage of listening sockets. So, to make it working, you will need to split up backend part and frontend part. Frontend will be looking into the database, which is filled by the backend. 

Are you sure you want to make things this complicated?

**Comment by onethings** [comment]
Yes, I mean just front end web ui mod deploy to app engine. Backend still use traccar server.

**Comment by vitalidze** [comment]
It does not look like it will be faster because it have to connect to some database running outside of the traccar. If you have slow connection between your database and appengine then the web UI will still be slow. From my point of view it should be much easier and cheaper to use a VPS. There is a wide range of VPS providers all over the world (starting from giants like amazon, google, microsoft azure) why don't you just choose one?

Anyway, you should search on how to deploy the java web application onto app engine.
